wherehows-kafka/src/main/java/wherehows/processors/MetadataLineageProcessor.java

```diff
@@ -76,26 +79,35 @@ private void processEvent(MetadataLineageEvent event) throws Exception {
     if (event.lineage == null || event.lineage.size() == 0) {
       throw new IllegalArgumentException("No Lineage info in record");
     }
-
     log.debug("MLE: " + event.lineage.toString());
 
-    agent appType = event.type;
-    if (whitelistAppTypes.size() > 0 && !whitelistAppTypes.contains(appType)) {
-      throw new RuntimeException("App Type not in whitelist, skip processing");
+    String actorUrn = null;
+    if (event.type != agent.UNUSED) {
```

Comment:
Maybe move all the actor logics into a separate function to improve readability?